modules/Hoot/managers/api.js

```diff
@@ -1428,6 +1428,30 @@ export default class API {
             } );
     }
 
+    getTimeoutTasks( projectId ) {
+        const params = {
+            path: `/grail/gettimeouttasks?projectId=${ projectId }`,
+            method: 'GET'
+        };
+
+        return this.request( params )
+            .then( resp => {
+                return {
+                    data: resp.data,
+                    message: 'Changeset upload complete.',
+                    status: 200,
+                    type: 'success'
+                };
+            } )
+            .catch( err => {
+                const message = err.data.message || 'Changeset upload failed.',
```

Comment:
We don't show a user message for this request so is it necessary to format the response as one?